cni-plugin/integration/run.sh

```diff
@@ -59,16 +71,11 @@ k run linkerd-proxy \
     --restart=Never \
     --rm \
     -- \
-    /usr/lib/linkerd/linkerd2-network-validator --log-format plain \
```

Comment:
this is probably an accidental deletion that is causing CI to fail.

Reply:
Re-added.